src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java

```diff
@@ -557,12 +558,8 @@ protected void registerRecipes(Consumer<IFinishedRecipe> consumer)
 		CombinationRecipeBuilder.of(MSBlocks.COARSE_STONE).input(Items.STONE).and().input(Items.GRAVEL).build(consumer);
 		CombinationRecipeBuilder.of(MSBlocks.CHISELED_COARSE_STONE).input(Items.CHISELED_STONE_BRICKS).and().namedInput(Items.GRAVEL).build(consumer);
 		CombinationRecipeBuilder.of(MSBlocks.CHISELED_COARSE_STONE).input(Items.CHISELED_STONE_BRICKS).and().namedInput(MSBlocks.COARSE_STONE).build(consumer);
-		CombinationRecipeBuilder.of(MSBlocks.SHADE_BRICKS).input(Items.STONE_BRICKS).or().namedInput(MSBlocks.BLUE_DIRT).build(consumer);
-		CombinationRecipeBuilder.of(MSBlocks.SHADE_BRICKS).input(Items.STONE_BRICKS).or().namedInput(Items.LAPIS_LAZULI).build(consumer);
-		CombinationRecipeBuilder.of(MSBlocks.SHADE_BRICK_STAIRS).input(Items.STONE_BRICK_STAIRS).or().namedInput(MSBlocks.BLUE_DIRT).build(consumer);
-		CombinationRecipeBuilder.of(MSBlocks.SHADE_BRICK_STAIRS).input(Items.STONE_BRICK_STAIRS).or().namedInput(Items.LAPIS_LAZULI).build(consumer);
```

Comment:
They were removed because of the addition of shade stone, which nullifies the reason for these recipes. It was comparatively out of place

Reply:
Can you clarify?
I get that `stone || blue dirt/lapis -> smooth shade stone` was removed in favor of `stone || blue dirt/dye-> shade stone`, but not the rest. (Though with the smooth stone block now in minecraft, it could've been changed to `smooth stone || blue dirt/dye -> smooth shade stone`)

Reply:
On second look, there are more examples of similar recipes to this than I realized. Adding these recipes back!